mappings/net/minecraft/client/render/block/entity/BeaconBlockEntityRenderer.mapping

```diff
@@ -1,4 +1,75 @@
 CLASS net/minecraft/class_822 net/minecraft/client/render/block/entity/BeaconBlockEntityRenderer
 	FIELD field_4338 BEAM_TEXTURE Lnet/minecraft/class_2960;
-	METHOD method_3543 render (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;FJII[F)V
-	METHOD method_3545 renderLightBeam (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_2960;FFJII[FFF)V
+	METHOD <init> (Lnet/minecraft/class_5614$class_5615;)V
+		ARG 1 ctx
+	METHOD method_22740 renderBeamFace (Lnet/minecraft/class_1159;Lnet/minecraft/class_4581;Lnet/minecraft/class_4588;FFFFIIFFFFFFFF)V
+		ARG 0 modelMatrix
+		ARG 1 normalMatrix
+		ARG 2 vertexConsumer
+		ARG 3 red
+		ARG 4 green
+		ARG 5 blue
+		ARG 6 alpha
+		ARG 7 yOffset
+		ARG 8 height
+		ARG 9 x1
+		ARG 10 z1
+		ARG 11 x2
+		ARG 12 z2
+		ARG 13 u1
+		ARG 14 u2
+		ARG 15 v1
+		ARG 16 v2
+	METHOD method_22741 renderBeamLayer (Lnet/minecraft/class_4587;Lnet/minecraft/class_4588;FFFFIIFFFFFFFFFFFF)V
+		ARG 0 matrices
+		ARG 1 vertexConsumer
+		ARG 2 red
+		ARG 3 green
+		ARG 4 blue
+		ARG 5 alpha
+		ARG 6 yOffset
+		ARG 7 height
+		ARG 8 x1
+		ARG 9 z1
+		ARG 10 x2
+		ARG 11 z2
+		ARG 12 x3
+		ARG 13 z3
+		ARG 14 x4
+		ARG 15 z4
+		ARG 16 u1
+		ARG 17 u2
+		ARG 18 v1
+		ARG 19 v2
+	METHOD method_23076 renderBeamVertex (Lnet/minecraft/class_1159;Lnet/minecraft/class_4581;Lnet/minecraft/class_4588;FFFFIFFFF)V
+		ARG 0 modelMatrix
+		ARG 1 normalMatrix
+		ARG 2 vertexConsumer
+		ARG 3 red
+		ARG 4 green
+		ARG 5 blue
+		ARG 6 alpha
+		ARG 7 y
+		ARG 8 x
+		ARG 9 z
+		ARG 10 u
+			COMMENT the left-most coordinate of the texture region
+		ARG 11 v
+			COMMENT the top-most coordinate of the texture region
+	METHOD method_3543 renderBeam (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;FJII[F)V
+		ARG 0 matrices
+		ARG 1 vertexConsumers
+		ARG 2 tickDelta
+		ARG 3 worldTime
+		ARG 5 yOffset
+		ARG 6 maxY
+		ARG 7 color
+	METHOD method_3545 renderBeam (Lnet/minecraft/class_4587;Lnet/minecraft/class_4597;Lnet/minecraft/class_2960;FFJII[FFF)V
+		ARG 0 matrices
+		ARG 1 vertexConsumers
+		ARG 2 textureId
+		ARG 3 tickDelta
+		ARG 5 worldTime
```

Comment:
Maybe these to fill the missing parameters on this method?
```
ARG 4 heightScale
ARG 7 beamSegment
ARG 8 segmentHeight
ARG 9 color
ARG 10 innerRadius
ARG 11 outerRadius
```

Reply:
Thank you, I was wondering what these params were